characters.txt

```diff
@@ -0,0 +1,26 @@
+Vanilla-visitor
+Andrew - Tax collector
+Steve - Groom
+Justin - Artisan
+Barry - Tinker
+
+Outlaws
+Evan - Stonemason
+Jon - Winemaker
+Mike - Blacksmith
+
+Andrew - Tax collector 
+Initial state
+- Outraged that you would delay him 
+
+1st talk option
+- Andrew bellows at you that he is a collector for the NTA (Nottingham Tax Association) and that you are costing the region valuable dollars by delaying him. If you continue, he demands recompense equal to his hourly wages.
+
+2nd talk option
+- Andrew demands to speak to your superior. While yelling, your superior comes over and confirms that Andrew is in fact a tax collector. He tells you not to waste anymore time and to let Andrew pass. 
```

Comment:
I'd prefer to leave this as a "If Andrew is still present when time ends, you get reamed out by the sheriff"

Reply:
Noted. I'll work that in.